### PR TITLE
Remove lorem text from card and popover examples

### PR DESCRIPTION
**Pull request by llorca**
have no idea what to do with the dialog example

### CODE REVIEW

**Comment by adidahiya** [comment]
let's use the Foundry snippet for dialog? https://www.palantir.com/palantir-foundry/

File: packages/docs-app/src/examples/core-examples/dialogExample.tsx [hunk 1]
```diff
@@ -16,34 +16,51 @@ export class DialogExample extends OverlayExample {
                 <Button onClick={this.handleOpen}>Show dialog</Button>
                 <Dialog
                     className={this.props.themeName}
-                    icon="inbox"
+                    icon="info-sign"
                     onClose={this.handleClose}
-                    title="Dialog header"
+                    title="Palantir Foundry"
```
Comment (giladgray) [inline packages/docs-app/src/examples/core-examples/dialogExample.tsx]: safe to use this name?

Comment (giladgray) [reply packages/docs-app/src/examples/core-examples/dialogExample.tsx]: yeah i suppose it comes from the website. cool stuff

File: packages/docs-app/src/examples/core-examples/cardExample.tsx [hunk 2]
```diff
@@ -26,10 +26,10 @@ export class CardExample extends BaseExample<ICardExampleState> {
         return (
             <Card {...this.state}>
                 <h5>
-                    <a href="#">Card heading</a>
+                    <a href="#">Announcement</a>
                 </h5>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-                <Button text="Submit" className={Classes.BUTTON} />
+                <p>The next major version of Blueprint is now available. Visit the documentation to learn more.</p>
```
[the remark this answers is not in the document]
Comment (adidahiya) [reply packages/docs-app/src/examples/core-examples/cardExample.tsx]: yeah I too think this should change. everything else looks good though

**Pull request merged**